Public/Src/Utilities/Native/IO/Unix/FileSystem.Unix.cs

```diff
@@ -816,7 +816,7 @@ private static Possible<ReparsePointType> GetReparsePointType(string path)
         {
             try
             {
-                FileAttributes attributes = TryGetFileAttributes(path);
+                FileAttributes attributes = File.GetAttributes(path);

```

Comment:
unrelated change: no need to call `TryGetFileAttributes` which catches/rethrows exceptions since the same is already done here.